src/code/z_game_over.c

```diff
@@ -13,8 +13,7 @@ void GameOver_FadeInLights(PlayState* play) {
     }
 }
 
-// This variable cannot be moved into this file as all of z_message_PAL rodata is in the way
-extern s16 gGameOverTimer;
+s16 gGameOverTimer = 0;
```

Comment:
I would just in-function-static this (and change to `s`) at this point, if its possible (not sure about its effects on bss ordering)

Reply:
It seems to affect regalloc actually

Reply:
Interesting, ok. Can just change the prefix to s since its local now